File: 1480_running_sum_of_1d_array.c
/**
 * Note: The returned array must be malloced, assume caller calls free().
 */
int* runningSum(int* nums, int numsSize, int* returnSize){
    
    int *retarray = (int *) malloc(sizeof(int) * numsSize);
    *returnSize = numsSize;

    for(int i=0; i<numsSize; i++)
    {
        retarray[i] = nums[i];
    }

    for(int i=1; i<numsSize; i++)
    {
        retarray[i] = retarray[i-1] + retarray[i];
    }
    
    
    return retarray;
}

File: 1480_running_sum_of_1d_array.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        for elements in range(1, len(nums), 1):
            nums[elements] = nums[elements - 1] + nums[elements]
        return nums            